code/modules/mob/living/carbon/superior_animal/human/subtype/eventwolf/eventwolf

```diff
@@ -0,0 +1,51 @@
+/mob/living/carbon/superior_animal/human/eventwolf
+	name = "Void Wolf Marauder"
+	desc = "A Void Wolf mercenary, heavily armoured with a large power-hammer."
+	melee_damage_lower = 60
+	melee_damage_upper = 70
+	icon_state = "voidwolfmarauder"
+	icon_dead = "voidwolfmarauder_dead"
+	health = 450
+	maxHealth = 450
+	ranged = FALSE
+
+
+	armor = list(melee = 50, bullet = 35, energy = 35, bomb = 0, bio = 100, rad = 50)
+	faction = "pirate"
+	icon = 'icons/mob/mobs-humanoid.dmi'
+
+/mob/living/carbon/superior_animal/human/eventwolf/handle_breath(datum/gas_mixture/breath) //we have are own air supplies
+	return
+
+/mob/living/carbon/superior_animal/human/eventwolf/handle_environment(var/datum/gas_mixture/environment) //are armor legit is a void suit
+	return
+
+/mob/living/carbon/superior_animal/human/eventwolf/start_pulling(var/atom/movable/AM)
+	to_chat(src, SPAN_WARNING("Your hand gets pushed away from \the [src]. !"))
+	return
+
+	get_stat_modifier = FALSE
+	breath_required_type = 0 // Doesn't need to breath, in a space suit
+	breath_poison_type = 0 // Can't be poisoned
+	min_air_pressure = 0 // Doesn't need pressure
+
+
+/mob/living/carbon/superior_animal/human/eventwolf/explosive
```

Comment:
Incredibly funny. are these tested as workiing?

Reply:
They use the same code as the boombas and other explosive enemies, though if tweaked a bit for health to ensure they can actually get in there and explode. (Take around 1-2 shots from an omnirifle before going down.)